guidelines/sample-server/files/models.json

```diff
@@ -81,5 +81,23 @@
         "format": "double"
       }
     }
+  },
+  "ErrorMessage": {
+    "description": "A shared model for error message",
+    "required" : ["error-code","reason"],
+    "properties": {
+      "error-code": {
```

Comment:
the purpose of this sample is to show how re-use works - not comply with best-practices for supporting different languages - just like other samples we have that show other features of Swagger without adhering to other "best practices"

That said - I'll happily change the name to something more to your liking :-)